include/nigiri/routing/search.h

```diff
@@ -136,17 +138,29 @@ struct search {
     state_.starts_.clear();
     add_start_labels(q_.start_time_, true);
 
+    auto processing_start_time = std::chrono::system_clock::now();
```

Comment:
```suggestion
    auto const processing_start_time = std::chrono::system_clock::now();
```

As it doesn't change.